test/fail_compilation/fail18372.d

```diff
@@ -0,0 +1,19 @@
+/*
+DISABLED: win32 win64 linux32 osx freebsd
```

Comment:
You've disabled this on every platform.

Reply:
Oh, this runs on linux64 only?

Reply:
The error occurs on macOS as well.

Reply:
It isn't really necessary to run it on all the posix platforms.